apps/prairielearn/src/server.ts

```diff
@@ -2216,6 +2227,17 @@ if ((esMain(import.meta) || (isHMR && !isServerInitialized())) && config.startSe
     // Load config immediately so we can use it configure everything else.
     await loadConfig(configPaths);
 
+    // Override the server port if PORT environment variable is set
```

Comment:
I figured this would be an easy way to plumb this variable over to the server from playwright. I could also read the `WORKER_INDEX` directly.